### ISSUE TITLE
Deauth works on phone, does not work on pc

### ISSUE THREAD

**Issue by vida1j** [sankethj/ZiFi#6]
When deauth is working only for phone after eding some code, it cannot deauth pc.
How do I fix it? Thanks

**Comment by sankethj** [comment]
include the part which u made changes to.

**Comment by vida1j** [comment]
> 

Yes, I include frame injection part - wifi_send_pkt_freedom and add some files like spacehuhnTech instruction. 
It can deauth phone, but cannot deauth pc.
Anyway, it is ok for now.
Are you planning to upgrade?

**Comment by sankethj** [comment]
I hv worked and made some changes and tested on other ESP based AP, may be I'll update here by February 15.

**Comment by vida1j** [comment]
It is really great news!
